src/wp-admin/options-discussion.php

```diff
@@ -162,7 +162,7 @@
 <?php _e( 'A comment is held for moderation' ); ?> </label>
 <br />
 <label for="wp_notes_notify">
-<input name="wp_notes_notify" type="checkbox" id="wp_notes_notify" value="1" <?php checked( '1', get_option( 'wp_notes_notify' ) ); ?> />
+<input name="wp_notes_notify" type="checkbox" id="wp_notes_notify" value="1" <?php checked( '1', get_option( 'wp_notes_notify', 1 ) ); ?> />
```

Comment:
Not a blocker: `checked` compare `'1'` string value with int 1. Do we needs to change this?

Reply:
Oh good catch, let me check if this should be and adjust.